src/components/Charts/WindTimeSeries.tsx

```diff
@@ -4,7 +4,7 @@
  */
 import Highcharts from "highcharts"
 import addWindBarbModule from "highcharts/modules/windbarb"
-import React from "react"
+import * as React from "react"
```

Comment:
Codacy found an issue: ['React' was used before it was defined.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6166252)